### PR TITLE
Erza MacGee, The Magical Trickster

### PR DESCRIPTION
**Pull request by ecowden**
H'okay: What did I mess up?

Also: now I wanna play... :grin: 

**Update**. Changed from fancy-schmancy semiauto pistol to a pair of revolvers, along with expanded backstory. The math says it's weaker, but it's who I wanted the character to be from the beginning. Screw the math! Be yourself!

'Cause really: who doesn't want to pretend to be a magical cowboy in a cyberpunk world?

### CODE REVIEW

**Comment by runicfox** [comment]
:+1: Well, if Lukas is a no go for tomorrow, we'll finish out character creation and I can get some prelude-level stuff done for you guys.  :)

File: shadowrunners/macgee.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+| Magical Suppliers    | 2      | Interest | Int       |
+| Magical Groups       | 2      | Interest | Int       |
+| Magical Theory       | 1      | Interest | Int       |
+| Magical Lore         | 2      | Interest | Int       |
+
+##Contacts
+
+| Name | Loyalty | Connection | Favor |
+| --- | --- | --- | --- | 
+| Talismonger | 3 | 4 | n/a |
```
Comment (runicfox) [inline shadowrunners/macgee.md]: The name of your Talismonger is Margie Dancing Crow.  She resides in and around the Renton/Salish-Shidhe Council border area.  Rumor has it that she knows both the fastest, and the quickest ways to cross that boundary without questions...

File: shadowrunners/macgee.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+| Street Vendor | 3 | 2 | n/a |
```
Comment (runicfox) [inline shadowrunners/macgee.md]: Charle Botha is a South African immigrant.  He always appreciates a heads up when Lone Star law enforcement might be sweeping through looking for reasons to crack skulls or do bribe shakedowns.

File: shadowrunners/macgee.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+| Homeless Person | 4 | 1 | n/a |
```
Comment (runicfox) [inline shadowrunners/macgee.md]: Gobber has perhaps slightly less than half his teeth.  The ones that remain appear only to stay affixed to his gums by sheer stubborn force.  Not unlike Gobber himself.  He may not have much (or anything, really), but you can always count on him to stash something for you, keep a secret, or play the insane drunkard when suits come asking after you.

File: shadowrunners/macgee.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+| Service Desk, Old Corp | 3 | 1 | n/a |
```
Comment (runicfox) [inline shadowrunners/macgee.md]: Sidney Bester ("Captain Sid Vicious" on the forums) tries to keep half an ear open for any chatter from the head office concerning you.  Level-1 help desk doesn't afford much in the way of real intel, but Sid says that if he hears anything, that he'll reach out to you.  Oh, and if you ever hear of a decent job that isn't tied to wage-slaving for your "dickhole parents", he's all ears.

**Comment by ecowden** [comment]
Ha! I love it! :smiling_imp:

**Pull request merged**